lib/paper_trail/config.rb

```diff
@@ -40,7 +40,8 @@ def track_associations
 
     # Indicates whether PaperTrail is on or off.
     def enabled
-      PaperTrail.paper_trail_store[:paper_trail_enabled].nil? || PaperTrail.paper_trail_store[:paper_trail_enabled]
+      PaperTrail.paper_trail_store[:paper_trail_enabled].nil? ||
+        PaperTrail.paper_trail_store[:paper_trail_enabled]
```

Comment:
The behavior here can more succinctly be written as:
```ruby
PaperTrail.paper_trail_store.fetch(:paper_trail_enabled, true)
```
Where the second argument, true, is used if nothing was set.